tests/socketcall_32bit/Makefile

```diff
@@ -0,0 +1,11 @@
+TARGETS=$(patsubst %.c,%,$(wildcard *.c))
+
+LDLIBS += -lpthread
+
+all: $(TARGETS)
+
+conn: conn.c
+	$(CC) $(CFLAGS) -o $@ $^
```

Comment:
True it does add an additional dependence, and potentially a problematic one as distros move away from distributing multilib applications.